Tasks/DownloadPackage/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "packagedownloader",
+  "name": "downloadpackage",
```

Comment:
It will show Package Id, we currently don't have support to show display value of picklist in instanceName